<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Windows" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>381</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_PVM"/>
    <addaction name="actionOpen_RAW"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionRotate_X"/>
    <addaction name="actionRotate_Y"/>
    <addaction name="actionRotate_Z"/>
    <addaction name="separator"/>
    <addaction name="actionScale_X"/>
    <addaction name="actionScale_Y"/>
    <addaction name="actionScale_Z"/>
    <addaction name="separator"/>
    <addaction name="actionTransfer_Function"/>
    <addaction name="actionReset_Function"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuTool"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_PVM">
   <property name="text">
    <string>Open PVM</string>
   </property>
  </action>
  <action name="actionOpen_RAW">
   <property name="text">
    <string>Open RAW</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRotate_X">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rotate X</string>
   </property>
  </action>
  <action name="actionRotate_Y">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Rotate Y</string>
   </property>
  </action>
  <action name="actionRotate_Z">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Rotate Z</string>
   </property>
  </action>
  <action name="actionScale_X">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Scale X</string>
   </property>
  </action>
  <action name="actionScale_Y">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Scale Y</string>
   </property>
  </action>
  <action name="actionScale_Z">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Scale Z</string>
   </property>
  </action>
  <action name="actionTransfer_Function">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Transfer Function</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionReset_Function">
   <property name="text">
    <string>Reset Function</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Windows</class>
   <extends>QWidget</extends>
   <header>windows.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
